hoomd/md/pytest/test_improper.py

```diff
@@ -7,8 +7,56 @@
                             autotuned_kernel_parameter_check)
 import pytest
 import numpy
+import math
 
 import itertools
+
+# these series of functions are used to calculate the values for the test cases used below
```

Comment:
These don't seem to be used below. I would suggest either using them in this method or preparing a separate `generate_periodic_improper_test_cases.py` file that uses them to generate the test cases used below.

Reply:
Now that they are called in the test, I'm fine with leaving them here. I would suggest wrapping them as static methods in a class to put them in a namespace that won't be confused with other tests:

```
class PeriodicImproperMath:
   @staticmethod
   def dchi_dr1(...)
```

Reply:
I added them to a separate file as suggested. I think this format is more clean.